### ISSUE TITLE
Room for text strings within the app is too limited, after localization data doesn't fit on screen

### ISSUE THREAD

**Issue by Meteor0id** [barbeau/gpstest#320]
I have attempted to keep the Dutch translations short, but sometimes a text string just needs some length in order to be informative. Is there any possibility of using the available space in a more tolerant matter?

![1](https://user-images.githubusercontent.com/34976212/62662315-f4be4a80-b973-11e9-8afc-0a3371775df9.png)
![2](https://user-images.githubusercontent.com/34976212/62662338-07d11a80-b974-11e9-8486-2052843f7395.jpeg)

**Comment by barbeau** [comment]
This is effectively a duplicate of https://github.com/barbeau/gpstest/issues/176, so let's talk more about configurable Status displays there.

In short, for the current Status view just abbreviate as much as you need to to get the info all on the screen, and explain the abbreviations in the Help text.

**Comment by paulle** [comment]
Making linebreaks and leaving the selection of the font size to the user would be a good solution.

**Comment by goldfndr** [comment]
If "E" in the accuracy labels is for Estimated then I suggest dropping it.

* It's relatively redundant — all of the accuracy values are estimates.
* It's not consistent — the speed and bearing accuracy labels don't include it.
* It can create confusion — the Help/About descriptions don't include it, so there's a mismatch.

**Comment by Meteor0id** [comment]
No, the E does mot stand for estimated. Please go to help > about to find tge well hidden legend which explains the way too short abbriviations.

**Comment by barbeau** [comment]
@goldfndr "E" does stand for "estimated" in the context of horizontal and vertical error.  I added this when I added the "Accuracy" feature to GPSTest to try and differentiate between the actual and estimated accuracy values.  In other words, I didn't want someone to set a ground truth value in Accuracy and then come back to Status and think that real accuracy was being displayed in Status.  Confusion between estimated and actual accuracy is a huge problem and this was my small attempt in a limited timeframe to address it.  I'd welcome better ideas for this in https://github.com/barbeau/gpstest/issues/176.

And you're right, this isn't in the Help/About section, so I need to update that.
